src/Bicep.Core/TypeSystem/TypeAssignmentVisitor.cs

```diff
@@ -636,8 +695,9 @@ private static void CollectErrors(List<ErrorDiagnostic> errors, ITypeReference r
         }
 
         private static TypeSymbol GetFunctionSymbolType(
-            FunctionCallSyntax syntax,
             FunctionSymbol function,
+            Token openParen,
```

Comment:
Since InstanceFunctionCallSyntax also calls this method, updated the signature to pass open and close paren which is what this method was using the syntax for. Probably we should consider InstanceFunctionCallSyntax inherit from FunctionCallSyntax but not sure about the implications of doing that.

Reply:
I'm good with leaving it as is. We can do that later if we want to.